connector/settings.gradle.kts

```diff
@@ -2,22 +2,35 @@ rootProject.name = "connector"
 
 pluginManagement {
     repositories {
+        mavenLocal()
+        maven {
+            url = uri("https://oss.sonatype.org/content/repositories/snapshots/")
```

Comment:
I observed that snapshot repository has been added as repository but I haven't seen any SNAPSHOT dependency used in the version catalog. maybe can this be removed?